cluster/config-defaults.yaml

```diff
@@ -1096,3 +1096,6 @@ sysctl_settings: ""
 
 # enables/disables the minDomains field for pod topology spread.
 min_domains_in_pod_topology_spread_enabled: "true"
+
+
+pod_deletion_cost_controller_memory: 200Mi
```

Comment:
Make sense to prefix with `skipper_`

Reply:
Was addressed by https://github.com/zalando-incubator/kubernetes-on-aws/pull/8001